corehq/apps/locations/templates/locations/manage/locations.html

```diff
@@ -77,15 +19,24 @@
     {% registerurl 'location_lineage' domain '-locid-' %}
     {% registerurl 'location_descendants_count' domain '-locid-'%}
 
+    {% registerurl 'edit_location' domain '-locid-' %}
```

Comment:
Looks like this is a bespoke version of `registerurl`'s `"---"` param, it'd be a minor enhancement to switch over to that.

Definitely not a blocking comment since you didn't introduce the `-locid-` business.